### DISCUSSION TITLE
How to add new menu to website navigation?

### DISCUSSION THREAD

**Discussion by GaneshKandu** [datenstrom/community#316] [Ask a question] [answered]
# want to add new menu to website
![img_20180328](https://user-images.githubusercontent.com/12833130/38024079-79616efe-3273-11e8-897b-df4ed6f1232c.png)
how i can do that didn't got any option in yellow ?

**Comment by BsNoSi** [comment]
You have to enumerate your content folders i.e.

1-want-to-add-more-menus-here
2-Home
3-About

That´s all.

You can change the menu title independent from the folder name (after Number-) with heading entry `TitleNavigation: Your Title here`

Good luck!

**Comment by GaneshKandu** [comment]
you means i need to do it manually every time when i need to create one more menu
is any option there to make one more menu after login in admin panel ?

**Comment by BsNoSi** [comment, marked as answer]
There is no administration panel. You setup your config-file, create a file folder structure, that´s it. This is a great advantage in compare to other CMS systems. You have only *files and folders*, see [documentation](https://developers.datenstrom.se/help/adding-content).

If you want to have a new/additional menu entry, you have to create a folder manually.
